Mathlib/CategoryTheory/Subobject/Basic.lean

```diff
@@ -558,6 +557,38 @@ theorem pullback_obj {X Y : C} (f : Y ⟶ X) (x : Subobject X) :
 
 instance (f : X ⟶ Y) : (pullback f).Faithful where
 
+lemma isPullback_aux (f : X ⟶ Y) (y : Subobject Y) :
+    ∃ φ, IsPullback φ ((pullback f).obj y).arrow y.arrow f := by
+  obtain ⟨A, i, ⟨_, rfl⟩⟩ := mk_surjective y
+  rw [pullback_obj]
+  exists (underlyingIso (pullback.snd (mk i).arrow f)).hom ≫ pullback.fst (mk i).arrow f
+  exact IsPullback.of_iso (IsPullback.of_hasPullback (mk i).arrow f)
+        (underlyingIso (pullback.snd (mk i).arrow f)).symm (Iso.refl _) (Iso.refl _) (Iso.refl _)
+        (by simp) (by simp) (by simp) (by simp)
+
+/-- `pullbackπ f y` is the first projection in the following pullback square:
```

Comment:
Could you make the docstring easier to understand out of context: this is what appears when we hover over this definition, so that we should understand this is about `Subobject.pullback`.